README.md

```diff
@@ -1209,6 +1210,16 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
   /(?:first|second)/
   ~~~
 
+* Use `Regexp#match?` when you don't use the captured result in Ruby.
```

Comment:
Do you think it's worth mentioning why?

```suggestion
* Use `Regexp#match?` when you don't use the captured result in Ruby, to avoid creating the more expensive `MatchData` object.
```